### PR TITLE
Web Application for American Sign Language Detection

### PR DESCRIPTION
**Pull request by Adhivp**
# Pull Request for DL-Simplified 💡

## Issue Title : [Feature Addition]: Web Application for American Sign Language Detection #540

- **Info about the related issue (Aim of the project)** : Web Application for American Sign Language Detection already created model
- **Name:** Adhithyan VP
- **GitHub ID:** Adhivp
- **Email ID:** adhivp910@gmail.com
- **Idenitfy yourself:**  GSSOC 2024

Closes: #540 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
- It was tested in my local system
Describe how have you verified the changes made

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Please attach the demo video in the `Web App/README.md` file. Just drag and drop the video file from your local system to README, it'll work.

One more thing, why you have used Inception instead of VGG as VGG is having better accuracy than Inception?
@Adhivp

**Comment by Adhivp** [comment]
It's the VGG model itself , when I saved the model output , the model output had inception name in it ,I will rename it 
Screenshot 2024-05-19 at 12 37 11 PM

**Comment by Adhivp** [comment]
While dragging and dropping the file is getting is saved in https://github.com/Adhivp/DL-Simplified/assets/92261845/aef089f7-19f2-410b-89de-ed5528511f47 , after merging it will be https://github.com/abhisheks008/DL-Simplified/assets/92261845/aef089f7-19f2-410b-89de-ed5528511f47 , i hope it is not a issue

**Comment by abhisheks008** [comment]
> While dragging and dropping the file is getting is saved in https://github.com/Adhivp/DL-Simplified/assets/92261845/aef089f7-19f2-410b-89de-ed5528511f47 , after merging it will be https://github.com/abhisheks008/DL-Simplified/assets/92261845/aef089f7-19f2-410b-89de-ed5528511f47 , i hope it is not a issue

Nopes it's not an issue.

**Comment by abhisheks008** [comment]
> It's the VGG model itself , when I saved the model output , the model output had inception name in it ,I will rename it Screenshot 2024-05-19 at 12 37 11 PM

Ahh my bad! The file name got me there.

**Comment by abhisheks008** [comment]
Ummm need to fine tune the backend of the web app a little bit. As in some cases it's not able to judge the signs.

**Comment by Adhivp** [comment]
No the model training is really bad , 
image
It just got 57% accuracy even in training data. 
validation accuracy is just 14%

**Comment by abhisheks008** [comment]
> No the model training is really bad , image It just got 57% accuracy even in training data. validation accuracy is just 14%

That's why it is not getting better results. Can you enhance the model?

**Comment by Adhivp** [comment]
issue #574 is already working on enanching model, we can merge this output model file for time being , after that work is done we can update the model?, what do you suggest?

**Comment by abhisheks008** [comment]
Let's wait for 2 more days, I hope that contributor will come up with some enhancement for sure.

**Comment by Adhivp** [comment]
Ok sure

**Comment by Adhivp** [comment]
It's been 2 days

**Comment by Adhivp** [comment]
@abhisheks008

**Comment by abhisheks008** [comment]
let me check with the other contributor @Adhivp

**Comment by abhisheks008** [comment]
Hi @Adhivp no updates as of now. That contributor is not responding. I have put a comment today also, let's see till EOD. If he doesn't reply at all, I'll unassign the issue and will look for other contributors for the same.

**Comment by Adhivp** [comment]
any updates?
